templates/hub/hub.css

```diff
@@ -62,6 +62,54 @@
   }
 }
 
+.hub-l1.appear {
```

Comment:
If whole page layout is required to be defined again then it makes more sense to create a new template. IMHO we must not send L2 CSS rules for L1 page (extra bytes)

Reply:
shall we move the specific sections to hub-l1.css and hub-l2.css?

Reply:
If most for L1 & L2 page has same layout then instead of having hub-l1 and hub-l2 as template we can just have hub as a template then the selector can be `.hub.appear` else we can have hub-l1.css and hub-l2.css
Maybe we will understand more in tomorrow's meeting after we go through L1 page structure.

Reply:
thanks, I will refactor this a bit to reduce repeated sections

Reply:
I extracted the common items, so the size is now reduced.